pkg/controllers/podgroup/pg_controller.go

```diff
@@ -76,7 +76,8 @@ func (pg *pgcontroller) Initialize(opt *framework.ControllerOption) error {
 	pg.podLister = pg.podInformer.Lister()
 	pg.podSynced = pg.podInformer.Informer().HasSynced
 	pg.podInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
-		AddFunc: pg.addPod,
+		AddFunc:    pg.addPod,
+		DeleteFunc: pg.deletePG,
```

Comment:
This is not good to call some time costing logic in eventhandler.

Reply:
Perhaps i can wirite the deleted pod in a queue,just like create pg

Reply:
> 

Perhaps i can wirite the deleted pod in a queue,just like create pg,do you think that's ok